platform/viewer/src/routes/index.tsx

```diff
@@ -20,6 +20,12 @@ const bakedInRoutes = [
     private: true,
     props: { children: WorkList },
   },
+  {
```

Comment:
A mode can be configured in the URL to use a custom data source, why you are adding another route for data source name? 

https://github.com/OHIF/Viewers/blob/v3-stable/platform/viewer/src/routes/buildModeRoutes.tsx#L53

Reply:
you can use `/tmtv/dicomjson` and `/tmtv/customDataSource` as a result of that

Reply:
Because there isn't a data source name route that allows showing the query page on a given data source.  Suppose I have two data sources, "default" and "aws" (as in the multiple.js configuration)  The first data source found would previously have defined the default worklist datasource, and there wouldn't be a way of querying to anything else.
Now, you can do: http://localhost:3000/aws to bring up a query page against the second data source.

Reply:
So I don't like breaking out convention that the first argument after `/` is the modeId. I believe having `/aws` and `/tmtv` is confusing. Can you put the data source name for the worklist as a param? something like `myviewer.com/search?datasource=aws` or similar?

Reply:
Changed so that the convention is follow - that is,
/search/:dataSourceName
for anything except the default data source name, which is still on /